lib/configuration_options.dart

```diff
@@ -0,0 +1,75 @@
+///
+///  Copyright © 2018-2019 PSPDFKit GmbH. All rights reserved.
+///
+///  THIS SOURCE CODE AND ANY ACCOMPANYING DOCUMENTATION ARE PROTECTED BY INTERNATIONAL COPYRIGHT LAW
+///  AND MAY NOT BE RESOLD OR REDISTRIBUTED. USAGE IS BOUND TO THE PSPDFKIT LICENSE AGREEMENT.
+///  UNAUTHORIZED REPRODUCTION OR DISTRIBUTION IS SUBJECT TO CIVIL AND CRIMINAL PENALTIES.
+///  This notice may not be removed from this file.
+///
+part of pspdfkit;
+
+/// Configuration options available can be set by the Flutter plugin.
+///
+/// See `main.dart` in the example project to check how they
```

Comment:
Feels like an early line wrapping when looking at the remaining line with used in the file.